sources/packages/api/src/route-controllers/application-exception/application-exception.aest.controller.ts

```diff
@@ -119,4 +125,30 @@ export class ApplicationExceptionAESTController extends BaseController {
       throw error;
     }
   }
+
+  /**
+   * Gets all pending student application exceptions.
+   * @param pagination options to execute the pagination.
+   * @returns list of pending student application exceptions.
+   */
+  @Get()
+  async getPendingApplicationExceptions(
+    @Query() pagination: ApplicationExceptionPaginationOptionsAPIInDTO,
+  ): Promise<PaginatedResultsAPIOutDTO<ApplicationExceptionSummaryAPIOutDTO>> {
+    const applicationExceptions =
+      await this.applicationExceptionService.getPendingApplicationExceptions(
+        pagination,
+      );
+
+    return {
+      results: applicationExceptions.results.map((eachApplication) => ({
```

Comment:
Just double-checking, case there are no records the value from `applicationExceptions.results` will be `null` or an empty array?

Reply:
An empty array will be returned. 


![image](https://user-images.githubusercontent.com/72251620/176307760-537b6381-1bca-476a-8809-9c4b380654cc.png)